x-pack/plugin/src/yamlRestTest/resources/rest-api-spec/test/esql/10_basic.yml

```diff
@@ -346,3 +346,27 @@ setup:
   - match: {values.0: ["1",2.0,null,true,123,123]}
   - match: {values.1: ["1",2.0,null,true,123,123]}
   - match: {values.2: ["1",2.0,null,true,123,123]}
+
+---
+lookup:
```

Comment:
This test is in the `lookup` PR and seems worth having. It doesn't run at the member, but it illustrates how you'd use this.